canonical-web-and-design/maas-ui
# feat(ui): add ability to edit logical volumes

## Done

- Built the "Edit logical volume" form

## QA

### MAAS deployment

To run this branch you will need access to one of the following MAAS deployments:

- [Karura](/HACKING.md#karura)
- [Bolla](/HACKING.md#bolla)
- [A development MAAS](/HACKING.md#development-deployment)
- [A local snap MAAS](/HACKING.md#snap-deployment) (this will not usually have machines)

### Running the branch

You can run this branch by:

- Serving with [dotrun](/HACKING.md#maas-ui-development-setup)
- [Building in a development MAAS](/HACKING.md#running-maas-ui-from-a-development-maas)

### QA steps

- Go to the storage tab of a machine in Ready or Allocated state
- If the machine does not have a logical volume, follow the steps in #1985 to create one
- Open the action menu for the logical volume and click "Edit logical volume..."
- Check that changing the tags and filesystem update correctly on submit

## Fixes

Fixes canonical-web-and-design/MAAS-squad#2289
